src/deflate/encode.rs

```diff
@@ -441,10 +429,7 @@ where
 mod tests {
     use super::super::Decoder;
     use super::*;
-    #[cfg(feature = "no_std")]
     use core2::io::{Read as _, Write as _};
-    #[cfg(not(feature = "no_std"))]
-    use std::io::{Read as _, Write as _};
```

Comment:
Not a change you made, but there is a `println` in one of the tests on line 473. This call makes `no_std` tests fail, please add:

```rust
#[cfg(feature = "std")]
println!("{:?}", finished.0);
```

Reply:
Done.